Sources/FoundationExtensions/AsyncExtensions.swift

```diff
@@ -69,4 +70,18 @@ internal enum Async {
         }
     }
 
+    /// Invokes a completion-block based API and returns the method `async`hronoiusly.
```

Comment:
I'm kinda lost as to how this'd be used, maybe we can add an example here to the docstring?

Reply:
I added examples for all methods in the file.